### DISCUSSION TITLE
Errors compiling with arduino_audio_tools with arduino_audio_driver

### DISCUSSION THREAD

**Discussion by Marco1971Repo** [pschatzmann/arduino-audio-tools#1553] [Q&A] [answered]
I' don't know if it is correct post here or it was better in issues anyway
I'm trying to compile player-url_icy-audiokit example in platformio.
```
lib_deps =
    https://github.com/pschatzmann/arduino-audio-tools.git#v0.9.8
    https://github.com/pschatzmann/arduino-audio-driver.git#v0.0.4
    https://github.com/pschatzmann/arduino-libhelix.git
```
And there is an error:
```
no matching function for call to audio_driver::AudioBoard::setMute(bool&, int&)
```

Effectively inAudioBoard,h:
```
bool setMute(bool enable) { return driver->setMute(enable); }
```
I'm doing something wrong or there is a misalignemente betweeb the libs?

**Comment by pschatzmann** [comment, marked as answer]
your requested audiotools version does not fit with the audio-driver version
